src/pages/dashboard/index.js

```diff
@@ -35,18 +37,35 @@ const Dashboard = () => {
   const getAllUsers = () => {
     getUsers().then(setUsers)
   }
-
+  
+  const getAllCohorts = () => {
+    getCohorts().then(setCohorts)
+  }
+  
   useEffect(getAllPosts, [])
   useEffect(getAllUsers, [])
-
-
+  useEffect(getAllCohorts, [])
+  
   const { openModal, setModal } = useModal()
-
+  
   const showModal = () => {
     setModal(`${t('createAPost')}`, <CreatePostModal getAllPosts={getAllPosts} />)
 
     openModal()
   }
+  
+  const shouldRenderCohortList = () => !!Array.isArray(cohorts)
```

Comment:
The `!!` shouldn't be needed here

Reply:
... because that method already returns a boolean value (*facepalms)